src/views/Player.vue

```diff
@@ -375,28 +374,32 @@ export default {
         .catch((err) => console.log(err));
 
       if (this.isTutorialUploadRequired) {
-        this.userConfigs["tutorial"]["isComplete"] = this.isTutorialComplete;
-        this.userConfigs["tutorial"]["progress"] = this.tutorialProgress;
-        const tutorial_progress = {
+        this.userConfig["tutorial"]["isComplete"] = this.isTutorialComplete;
+        this.userConfig["tutorial"]["progress"] = this.tutorialProgress;
+        const userConfig = {
           "user-id": this.userId,
-          configs: this.userConfigs,
+          configs: this.userConfig,
```

Comment:
Continuing renaming, and name `configs` -> `config` also?

Reply:
The only issue here is that we'll have to go and rename all user config jsons. This is the default structure of the user config - I had initially also proposed keeping `config` here but deepansh felt that because this contains multiple configs like for tutorial progress, experiment allocation and now, locale, it should be plural.

Reply:
Then maybe keep the other one also as plural? Now we are both inconsistent and incorrect :D Better to be one of the two :D

Reply:
achcha got it. makes sense.